samples/CommunityToolkit.Maui.Sample/Platforms/Android/MainActivity.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace CommunityToolkit.Maui.Sample;
 
-[Activity(Theme = "@style/Maui.SplashTheme", MainLauncher = true, LaunchMode = LaunchMode.SingleTask, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation | ConfigChanges.UiMode | ConfigChanges.ScreenLayout | ConfigChanges.SmallestScreenSize)]
```

Comment:
Is `ResizeableActivity =true` recommended for all devs consuming `MediaElement`? 

If so, let's add this to the docs (CC @bijington)

Reply:
Yes it is recommended. This will allow split screen and multi window. So with media Element this lets you dynamically resize the window and allows for the option later to add PIP support.

Reply:
Ok great! I'll mark this PR as approved, but could you open a quick PR updating the docs here to add `ResizeableActivity = true` as a requirement for Android? Once the docs PR is open, I'll merge this 👍

Reply:
Here is the PR for docs. https://github.com/MicrosoftDocs/CommunityToolkit/pull/412